File: scrap/5.1/MailService.java
package popa.catalin.scrap;

public class MailService {

    public void sendMail(String msg) throws InterruptedException {
        System.out.printf("Sending email {%s} \n", msg);
        Thread.sleep(1000);
        System.out.println("Email sent.");
    }
}

File: Laboratoare_gbir2109/DP_Lab5_Homework/Worker/MailService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;

namespace Worker
{
    class MailService
    {
        public void SendMail(String msg)
        {
            Console.WriteLine($"Sending email {msg} \n");
            Thread.Sleep(1000);
            Console.WriteLine("Email sent.");
        }
    }
}
